src/cljc/rems/common/form.cljc

```diff
@@ -464,6 +464,19 @@
                    empty-label
                    nil-label))))))
 
+    ;; (testing "phone number"
+    ;;   (let [form (assoc form :form/fields
+    ;;                     [{:field/type :phonenumber
+    ;;                       :field/title {:en "en" :fi "fi"}
+    ;;                       :field/optional false}])]
+    ;;     (testing "valid form"
+    ;;       (is (empty? (validate-form-template form languages))))
+
+    ;;     (testing "missing title localization"
+    ;;       (is (= {:form/fields {0 {:field/title {:en :t.form.validation/required}}}}
+    ;;              (validate-form-template (assoc-in form [:form/fields 0 :field/title :en] "") languages)
+    ;;              (validate-form-template (assoc-in form [:form/fields 0 :field/title :en] nil) languages))))))
```

Comment:
A test like this would be nice. Remember to also test a valid non-empty value and an invalid non-empty value.